src/main/java/com/openlattice/chronicle/constants/EdmConstants.java

```diff
@@ -11,51 +11,57 @@ private EdmConstants() {
-    public static final FullQualifiedName MODEL_FQN               = new FullQualifiedName( "vehicle.model" );
+
+    public static final FullQualifiedName ACTIVE_FQN              = new FullQualifiedName( "ol.active" );
+    public static final FullQualifiedName DATE_LOGGED_FQN         =  new FullQualifiedName( "ol.datelogged" );
     public static final FullQualifiedName DATE_TIME_FQN           = new FullQualifiedName( "ol.datetime" );
-    public static final FullQualifiedName START_DATE_TIME         = new FullQualifiedName( "ol.datetimestart" );
+    public static final FullQualifiedName DATETIME_FQN            = new FullQualifiedName( "ol.datetime" );
+    public static final FullQualifiedName DURATION_FQN            = new FullQualifiedName( "general.Duration" );
     public static final FullQualifiedName FULL_NAME_FQN           = new FullQualifiedName( "general.fullname" );
-    public static final FullQualifiedName TITLE_FQN               = new FullQualifiedName( "ol.title" );
-    public static final FullQualifiedName RECORD_TYPE_FQN         = new FullQualifiedName( "ol.recordtype" );
-    public static final FullQualifiedName DURATION                = new FullQualifiedName( "general.Duration" );
+    public static final FullQualifiedName GENERAL_ID_FQN          = new FullQualifiedName( "general.stringid" );
+    public static final FullQualifiedName MODEL_FQN               = new FullQualifiedName( "vehicle.model" );
     public static final FullQualifiedName OL_ID_FQN               = new FullQualifiedName( "ol.id" );
-    public static final FullQualifiedName USER_FQN                = new FullQualifiedName( "ol.user" );
+    public static final FullQualifiedName PERSON_ID_FQN           = new FullQualifiedName( "nc.SubjectIdentification" );
+    public static final FullQualifiedName RECORD_TYPE_FQN         = new FullQualifiedName( "ol.recordtype" );
+    public static final FullQualifiedName START_DATE_TIME_FQN     = new FullQualifiedName( "ol.datetimestart" );
+    public static final FullQualifiedName STATUS_FQN              = new FullQualifiedName( "ol.status" );
+    public static final FullQualifiedName STRING_ID_FQN           = new FullQualifiedName( "general.stringid" );
     public static final FullQualifiedName TIMEZONE_FQN            = new FullQualifiedName( "ol.timezone" );
+    public static final FullQualifiedName TITLE_FQN               = new FullQualifiedName( "ol.title" );
+    public static final FullQualifiedName USER_FQN                = new FullQualifiedName( "ol.user" );
+    public static final FullQualifiedName VERSION_FQN             = new FullQualifiedName( "ol.version" );
     public static final FullQualifiedName VALUES_FQN              = new FullQualifiedName( "ol.values" );
     public static final FullQualifiedName COMPLETED_DATE_TIME_FQN = new FullQualifiedName( "date.completeddatetime" );
-    public static final FullQualifiedName ACTIVE_FQN              = new FullQualifiedName( "ol.active" );
```

Comment:
I like that these are alphabetized now but I hate that I can't tell which ones are new. Which ones are new?

Reply:
```
removed:
- NOTIFICATION_ENTITY_SET_PREFIX


added:
- DICTIONARY_ENTITY_SETNAME
- PERSON_FQN
- GENERAL_ID_FQN
- PERSON_ID_FQN

changes:
- DURATION -> DURATION_FQN
- START_DATE_TIME -> START_DATE_TIME_FQN
```

Reply:
👍